zen/plugins/loader.py

```
@@ -59,42 +61,42 @@ async def load_plugin_from_git(self, git_url: str, version: str = "main") -> Opt
-            
+
             # Reload manifest
             manifest = await self._load_manifest(entry.local_path)
             if not manifest:
                 print(f"Failed to reload manifest for {plugin_id}")
                 return False
-            
+
             # Update registry entry
             entry.manifest = manifest
             entry.last_updated = datetime.now()
```

Ruff (0.14.10)

Undefined name `datetime`

(F821)